Tensile/Source/client/include/ResultFileReporter.hpp

```diff
@@ -79,6 +79,9 @@ namespace Tensile
             int64_t     m_winnerSolutionIdx = -1;
             double      m_fastestGflops     = -1.0;
             double      m_fasterTimeUS      = -1.0;
+            // for logging winner GFx clock in CSV file
+            uint16_t m_currGfxClock   = 0;
```

Comment:
Could you keep the alignment?

Reply:
clang-format did that alignment, I  modified to match the exisiting code.

Reply:
> clang-format did that alignment, I modified to match the exisiting code.

OK. Then, please follow clang-format.